server/src/test/helper.ts

```diff
@@ -58,3 +60,21 @@ export function sleep(ms: number): Promise<unknown> {
     setTimeout(resolve, ms);
   });
 }
+
+export async function shouldProvideCompletions(
+  completionProvider: DbtNodeCompletionProvider,
+  text: string,
+  expectedCompletions: { label: string; insertText: string }[],
+): Promise<void> {
+  const completions = await completionProvider.provideCompletions(text);
+  assertCompletions(completions, expectedCompletions);
+}
+
+function assertCompletions(actualCompletions: CompletionItem[] | undefined, expectedCompletions: { label: string; insertText: string }[]): void {
+  assertThat(actualCompletions, defined());
+  actualCompletions?.forEach((actualItem, i) => {
```

Comment:
Can we use `contains` or `containsInAnyOrder`?

Reply:
Seems complicated.